features/step_definitions/assertion_steps.rb

```diff
@@ -0,0 +1,11 @@
+Then("I should see a navbar") do
+    expect(page).to have_css '#navbar'
+end
+
+Then("I should see {string}") do |content|
+    expect(page).to have_content content
+end
+
+Then("show me the page") do
```

Comment:
This isn't an assertion - I'd suggest we move this to a new file called something like "helper_steps.rb".

Reply:
fixed